src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -133,7 +133,14 @@ void searchConcepts(CloudPlatform cloudPlatform) {
     when(datasetService.retrieve(dataset.getId())).thenReturn(dataset);
     SnapshotBuilderSettings snapshotBuilderSettings =
         new SnapshotBuilderSettings()
-            .domainOptions(List.of(new SnapshotBuilderDomainOption().category("condition").id(19)));
+            .domainOptions(
+                List.of(
+                    (SnapshotBuilderDomainOption)
```

Comment:
The casts are necessary because the chained constructors lose the types along the way. Given this is test data it seemed reasonable to have them